tests/unit/disco/test_utils.js

```diff
@@ -1,14 +1,39 @@
-import { sanitizeHTMLWithExternalLinks } from 'disco/utils';
-
-describe('sanitizeHTMLWithExternalLinks', () => {
-  it('adds `target` and `rel` attributes to HTML "targetable" elements', () => {
-    const html = '<a href="http://example.org">link</a>';
-    expect(sanitizeHTMLWithExternalLinks(html, ['a'])).toEqual({
-      __html: [
-        '<a href="http://example.org" target="_blank" rel="noopener noreferrer">',
-        'link',
-        '</a>',
-      ].join(''),
+/* eslint camelcase: 0 */
+import {
+  makeQueryStringWithUTM,
+  sanitizeHTMLWithExternalLinks,
+} from 'disco/utils';
+
+describe(__filename, () => {
+  describe('sanitizeHTMLWithExternalLinks', () => {
+    it('adds `target` and `rel` attributes to HTML "targetable" elements', () => {
+      const html = '<a href="http://example.org">link</a>';
+      expect(sanitizeHTMLWithExternalLinks(html, ['a'])).toEqual({
+        __html: [
+          '<a href="http://example.org" target="_blank" rel="noopener noreferrer">',
+          'link',
+          '</a>',
+        ].join(''),
+      });
+    });
+  });
+
+  describe('makeQueryStringWithUTM', () => {
+    it('returns a valid query string', () => {
+      const utm_content = 'utm-content';
+      const queryString = makeQueryStringWithUTM({ utm_content });
+
+      expect(queryString).toEqual(
+        `?utm_source=discovery.addons.mozilla.org&utm_medium=firefox-browser&utm_content=${utm_content}`,
```

Comment:
Because `makeQueryStringWithUTM` internally calls `makeQueryString` from `core/api`, we should probably also call that as part of the expectation. We know, for now, it's not going to return anything other than what we expect here, but `makeQueryStringWithUTM` doesn't know that, and this test is now relying on `makeQueryString` never changing.